packages/v4/src/content/design-guidelines/components/modal/modal.md

```diff
@@ -7,111 +7,176 @@ related: ['Button']
 
-**Note:** If information is not critical, don’t use a passive dialog. Instead, use inline notifications or toast notifications.
+When a text input field is added to a confirmation dialog, the danger button will only become enabled once a user enters the entire word or phrase.
 
-#### Get required user input
-Sometimes users may need to provide additional input in order to complete an action. In this situation a form or other type of element may be incorporated into the modal. An example of this use case is the [modal wizard](/components/modal/react#with-wizard).
+<img src="./img/destructive-confirmation-dialog-steps.png" alt="A multi-step destructive confirmation dialog's button activates only after a user types DELETE into the input field" />
 
-### Content
-* Clearly restate the action the user is trying to take and explicitly outline the consequences of that action.
 
-* Provide button labels that make it simple for users to understand and confirm their choices. Use descriptive verbs that clearly indicate the consequence of a selection, like Delete, Save, or Create. Avoid vague labels like OK or Done.
+### Error dialogs
 
-* Use descriptive title and button labels so the user can scan content and make a decision quickly. Ambiguous title and button labels make it more difficult for the user to understand their options and make a decision.
+<img src="./img/error-dialog.png" alt="An example of a successful error dialog, with and without an error icon" />
 
-* If the modal needs to convey the importance of information visually, icons can be added.
+Use an error dialog to inform users of problems that interrupt normal or expected behavior. Briefly contextualize the problem and why it happened, then provide actionable steps toward a solution.
+
+Learn more about writing error messages in our [error message guidelines](/ux-writing/error-messages).
+
+Error dialogs may use an error icon for visual emphasis.
+
+### Passive dialogs
+
+<img src="./img/passive-dialog.png" alt="An example of a successful passive dialog, with and without a warning icon" />
+
+Use a passive dialog to communicate critical and immediately relevant information like the status of an application or an action’s result. Passive dialog content should require or lead to user action.
+
+Passive dialogs can also notify the user about the status or duration of a critical process, especially if the process may take an unusual amount of time to complete.
+
+Never use a passive dialog for non-critical information. Instead, use an [inline alert](/components/alert/design-guidelines/#inline-alerts) or a [toast alert](/components/alert/design-guidelines/#toast-alerts).
+
+### Using dialogs to prompt user input
```

Comment:
You say there are 3 types of modals, but this seems like a fourth type? Which actually when I read the 3 types, made me think there was one missing which I think is what this is. Basically the use case where modals are used as a from, or used for user input.

Reply:
Does "Input dialogs" make sense to title this modal type in alignment with the three other types?

Reply:
Yes, I think so!! And then def add it to the list when you first introduce the types